Update clientonly.txt

NoMoreUsernames999 opened the pull request:
Added:

Dynamic darkness.
Inventory Move.
Game stage viewer.
Equipment Compare.
Better Chat.
Better Foliage.
Better Ping Display.
Legendary Tooltips.
(Chat) Ping!
Smart Hud.
Custom Loading Screens.
(Menu) Blur.
Dynamic Surroundings (+ Hud).
Enhanced Visuals. 
Apple Skin.
Biome Biome Blender.
Better Third Person.
Block Overlay Fix.
Simple Discord RPC.
Conflict In Harmony. 
Journey Map Stages.
JEI Resources.
Mob Dismemberment.
No More Recipe Book.
Nothirium.
Iceberg Core.
Render Library. 
More Default Options.
Client Tweaks.
Smooth Chat Flow.
Modpack Configuration Checker.
Difficulty Lock.
Better Book display.
Controlling
Better Advancements.
Better Hud.
Custom Nausea (Discontinued).
Biome Border Viewer.
Connected Textures Mod.
Just Enough Throwing In Fluids.
Particle Culling.
Malek's Multiblock Mobs.
No Night Vision Flashing.

NoMoreUsernames999 committed:
Update clientonly.txt

Added:

Dynamic darkness.
Inventory Move.
Game stage viewer.
Equipment Compare.
Better Chat.
Better Foliage.
Better Ping Display.
Legendary Tooltips.
(Chat) Ping!
Smart Hud.
Custom Loading Screens.
(Menu) Blur.
Dynamic Surroundings (+ Hud).
Enhanced Visuals. 
Apple Skin.
Biome Biome Blender.
Better Third Person.
Block Overlay Fix.
Simple Discord RPC.
Conflict In Harmony. 
Journey Map Stages.
JEI Resources.
Mob Dismemberment.
No More Recipe Book.
Nothirium.
Iceberg Core.
Render Library. 
More Default Options.
Client Tweaks.
Smooth Chat Flow.
Modpack Configuration Checker.
Difficulty Lock.
Better Book display.
Controlling
Better Advancements.
Better Hud.
Custom Nausea (Discontinued).
Biome Border Viewer.
Connected Textures Mod.
Just Enough Throwing In Fluids.
Particle Culling.
Malek's Multiblock Mobs.
No Night Vision Flashing.

CalaMariGold commented:
A couple of these are already in here, can you go through and remove the duplicates? Thank you!

NoMoreUsernames999 commented:
~~Odd, I used the page search function to insure I wasn't adding any duplicates...~~

Derp moment. Should be fixed now...

NoMoreUsernames999 committed:
Update clientonly.txt

Hopefully fix all duplicates. Checked every entry with *in file search* instead of page search this time >_<.
